server/controllers/wip/questionsController.ts

```diff
@@ -1,3 +1,111 @@
+import { FormWizard } from 'hmpo-form-wizard'
+import type express from 'express'
 import { BaseController } from '../index'
+import { ReportWithDetails } from '../../data/incidentReportingApi'
+import format from '../../utils/format'
+import { getIncidentTypeConfiguration } from '../../reportConfiguration/types'
+import {
+  AnswerConfiguration,
+  IncidentTypeConfiguration,
+  QuestionConfiguration,
+} from '../../data/incidentTypeConfiguration/types'
+import logger from '../../../logger'
 
-export default class QuestionsController extends BaseController {}
+export default class QuestionsController extends BaseController<FormWizard.MultiValues> {
+  getBackLink(_req: FormWizard.Request, _res: express.Response): string {
+    return '/reports/'
+  }
+
+  middlewareLocals(): void {
+    this.use(this.lookupReport)
+    super.middlewareLocals()
+  }
+
+  async lookupReport(req: FormWizard.Request, res: express.Response, next: express.NextFunction): Promise<void> {
+    const { incidentReportingApi } = res.locals.apis
+    const reportId = req.params.id
+
+    res.locals.report = await incidentReportingApi.getReportWithDetailsById(reportId)
```

Comment:
the wrapping router already called `.getReportById`… should it instead have called this function instead so we don't need 2 lookups?

Reply:
I noticed there was already a middleware that was doing the report lookup so I've tweaked things a little to reuse that:
- `populateReport` middleware gets the report with Details and also the report config now: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/269/commits/1bbb40c9ea251e31d93e4c8ff3c5b7a0f2d5c783
- Removed logic to get report/config from questions route/controller: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/269/commits/2df7b7d8dbd9eed24fdec518300849ea226c27df